meta/send.rb

```diff
@@ -307,6 +307,68 @@
   mutation 'foo.__send__(self)'
 end
 
+Mutant::Meta::Example.add :send do
+  source '__send__(:foo)'
+
+  singleton_mutations
+  mutation 'foo'
+  mutation '__send__'
+  mutation 'public_send(:foo)'
+  mutation ':foo'
+  mutation '__send__(nil)'
+  mutation '__send__(self)'
+  mutation '__send__(:foo__mutant__)'
+end
+
+Mutant::Meta::Example.add :send do
+  source 'foo.send(:bar)'
+
+  singleton_mutations
+  mutation 'foo.bar'
+  mutation 'foo.send'
+  mutation 'foo.__send__(:bar)'
+  mutation 'foo.public_send(:bar)'
+  mutation 'foo'
+  mutation ':bar'
+  mutation 'self.send(:bar)'
+  mutation 'foo.send(nil)'
+  mutation 'foo.send(self)'
+  mutation 'foo.send(:bar__mutant__)'
+end
+
+Mutant::Meta::Example.add :send do # rubocop:disable Metrics/BlockLength
+  source 'foo.public_send(:bar, 1, two: true, **kwargs, &block)'
```

Comment:
We need to prove that we forward "all kinds of arguments" hence yes this test case is "warranted". Even if it exposes us to secondary mutations in this tests.

I was thinking about a strategy to filter the secondaries, but it depends on making the AST matching language. for now we have to eat it.

Reply:
#415 Might also be a good partial solution if you would like me to investigate adding something like that.